build/packages-template/bbb-config/after-remove.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash -e
+
+case "$1" in
+   remove|failed-upgrade|abort-upgrade|abort-install|disappear)
+   ;;
+   purge)
+      # remove file deployed by after-install script if it is still a symlink
+      if [ "x$(readlink -f /etc/bigbluebutton/nginx/include_default.nginx)" = x/usr/share/bigbluebutton/include_default.nginx ] ; then
```

Comment:
Not sure if we should introduce the x-prefix idiom here. It's not used anywhere else and coverse rare edge cases (a value starting in `-` being interpreted as an operator, for those who wonder). I'd rather quote the RHS.